### ISSUE TITLE
Importing a lot of empty albums

### ISSUE THREAD

**Issue by rdybro** [electerious/Lychee#469]
I got a question - I am experiencing some inconsistencies in my imports of photos into Lychee. I don't think it is a particular problem with Lychee, more like a problem with me ;-)

What I have done is I have uploaded some folders with photos inside to the webserver running Lychee, and then made a Import from server.

First I uploaded one folder to "/mnt/test/album1" and then made a Import from "/mnt/test/". This imported all my photos in one album, but also created one empty album for every image in the folder.

Then I uploaded another folder to "/mnt/test/album2" and imported "/mnt/test/album2". This created an empty album for all the images inside that folder, and imported the photos as Unsorted.

I have a lot of these errors in my log:
2016-03-03 20:43:45 - error - Import::server (187) - Could not import folder. Function returned warning.

Can any of you maybe tell me a little about what I am doing wrong? Thanks in advance :) Regards

**Comment by electerious** [comment]
Hi @rdybro, can you post the output of the log (Settings -> Show Log)? Should tell us more about the warning.

**Comment by rdybro** [comment]
Sure - will do that later when I get access to the machine again :) But as far as I remember it only contains a lot of lines identical with the one I posted before, but I am not sure about that, so I will post.

Thanks :)

**Comment by electerious** [comment]
Oh sorry, just realized you already attached a line from the log. I should improve the log messages :0 "Could not import folder. Function returned warning." isn't really helpful.

**Comment by electerious** [comment]
The log message means that the folder only contained albums. Looks like PHP/Lychee thinks that the photos *are* albums. What kind of photos do you try to upload? Is it possible to upload them via the normal file upload or will this fail, too?

**Comment by rdybro** [comment]
I don't suppose there would be any problems importing file by file. I will test it when I come home.

The reason I did the import of the folders was because I hoped I could get around uploading all my photos one by one :)

The weird things is though, that all the images was imported, but you explanation makes sense in the matter of Lychee making all these empty albums.

All my photos I uploaded was taken by me from my smart phone. All the images are named like this: "IMG_20140518_224926". The folders containing the images was named like "YYYY-MM-DD - Event name". Could there be a problem in the namings?

**Comment by rdybro** [comment]
I have just tested. If I upload just a single photo through the upload function, it works as expected. It uploads the photo and imports it as unsorted, and no empty albums is created.

**Comment by electerious** [comment]
No idea why this happens, sorry :-/

**Comment by rdybro** [comment]
Now I feel bad. A couple of days after posting this, I actually found the cause of the problem, I just forgot all about writing in here :-( I am truly sorry about that.

As far as I remember, the problem was that the folder I had copied to the Lychee machine, and tried to import had previously been used with a cloud syncing software. This software had made a lot of hidden folders, and it was these Lychee created as empty albums.

So nothing is broken, quite opposite in fact, it seems to work as intended ;-)

Again, I am sorry I didn't update in here :-(
